preview-src/index.ts

```diff
@@ -161,7 +160,10 @@ if (settings.scrollEditorWithPreview) {
       scrollDisabled = false
     } else {
       const line = getEditorLineNumberForPageOffset(window.scrollY)
-      if (typeof line === 'number' && !isNaN(line)) {
+      if (window.scrollY === 0) {
+        // scroll to top, document title does not have a data-line attribute
+        messaging.postMessage('revealLine', { line: 1 })
```

Comment:
I think lines are indexed beginning at `0` :stuck_out_tongue_winking_eye: 

When I had `1` My source document line stopped at line `2` when I was at the top of the preview.

We gaze upon our tooltip assistance and see it is zero-indexed:

![image](https://user-images.githubusercontent.com/674783/140018697-40e87058-b0b0-4d8d-9232-0b856c164458.png)

Reply:
```suggestion
        messaging.postMessage('revealLine', { line: 0 })
```